src/lightning/pytorch/loggers/mlflow.py

```diff
@@ -151,6 +157,7 @@ def __init__(
         from mlflow.tracking import MlflowClient
 
         self._mlflow_client = MlflowClient(tracking_uri)
+        self._logged_metrics = set()  # Track (key, step)
```

Comment:
Probably need to import `Set` and `Tuple` from typing
```suggestion
        self._logged_metrics: Set[Tuple[str, float]] = set()  # Track (key, step)
```